.editorconfig

```diff
@@ -1,7 +1,12 @@
 root = true
+
 [*]
+charset = utf-8
 end_of_line = lf
 insert_final_newline = true
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
In Markdown, two spaces in the end of a line means line-break.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lines